README.md

```diff
@@ -1,6 +1,6 @@
 # DataHub: A Generalized Metadata Search & Discovery Tool
 [![Version](https://img.shields.io/github/v/release/linkedin/datahub?include_prereleases)](https://github.com/linkedin/datahub/releases)
-[![Build Status](https://travis-ci.org/linkedin/datahub.svg)](https://travis-ci.org/linkedin/datahub)
+![Gradle Build & Test](https://github.com/linkedin/datahub/workflows/Gradle%20Build%20&%20Test/badge.svg)
```

Comment:
For consistency with other badges consider renaming the workflow to simply "build" or "build & test"?

Reply:
Done